WISE-Community/WISE-Client
# fix(Contact WISE): Form shows recaptcha error when not logged in

## Changes

Changed the Contact WISE form to use the recaptcha V3 method of verifying. It used to use the V2 method.

## Test

- When not logged in
  - make sure you can submit the Contact WISE form
  - make sure the recaptcha error message shows up when it should. You can trigger this by setting a breakpoint in the ContactFormComponent.submit() function and changing the recaptchaResponse value to a string like 'abc'
- Make sure the Contact WISE form works when recaptcha is not enabled. You can do this by clearing out the recaptcha keys in application.properties (recaptcha_public_key and recaptcha_private_key).

Closes #1309
